types/google.picker/index.d.ts

```diff
@@ -190,6 +190,7 @@ declare namespace google {
             setMimeTypes(mimeTypes: string): DocsView;
 
             // Allows the user to select folders from Shared team Drives.
+            /** @deprecated - use setEnableDrives instead */
```

Comment:
Should this be google.picker.DocsView.setEnableDrives to match the other comment?

Reply:
I thought about that as well, the reason they're different is because as an api consumer you would only use `google.picker.Feature.SUPPORT_DRIVES`, but `setEnableDrives` would be chained on the previous call like:

```
new google.picker.DocsView(google.picker.ViewId.DOCS).setEnableTeamDrives(true)
```

My thinking was that this would be less confusing to the user who saw the deprecation message and was wondering how to migrate.

Reply:
Makes sense. Thanks for the clarification.